content/docs/06-integrations/00-aws-ebs.md

```diff
@@ -32,6 +32,219 @@ AWS Elastic Block Store is an easy to use, high performance block storage at any
 </Tabs.TabPane>
 </Tabs>
 
+# Pre-requisite Permissions
+
+1. [AmazonEBSCSIDriverPolicy](/integrations/aws-ebs#amazonebscsidriverpolicy) the minimum permission to be attached.
+
+
+2. [EBSCSIKMSEncryptionPolicy](/integrations/aws-ebs#ebscsikmsencryptionpolicy) (custom policy name), if the user wants to enable KMS Key encryption.
+
+
+3. For EKS cluster, make the following modifcation to the Kubernetes values.yaml file attached to the cluster profile:
```

Comment:
this is valid if user doesn't want to add these permissions to aws cloud account 

then he will need to do this 

otherwise only adding policies at aws cloud account level will take care of both aws infra and eks automatically